pyproject.toml

```diff
@@ -8,8 +8,14 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
+requests = "^2.25.0"
+pytest = "^6.2.2"
```

Comment:
This seems to be missing `python-dateutil` that your code depends on - the tests errored for me until I ran `poetry add python-dateutil`

Reply:
Done